_posts/blog/2019-05-20-Ruslan-GSoC.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Ruslan GSoC Student
+categories: blog
+excerpt: A new member of the Whitaker lab
+image:
+  feature:
+link:
+date: 2019-05-20
+modified:
+share: true
+author: ruslan_yermakov
+---
+
+## TL;DR 
+
+> My name is Ruslan Yermakov, I am a senior undergraduate student with a major in Computer Science at Sumy State University, Ukraine.
+I find it so hard to put into words how much I am enthusiastic to be accepted as a Google Summer of Code student 
+and excited to become a member of the Whitaker lab research group and to contribute to the scona project, an open-source toolkit for neuroscientists!
+My interests include data visualization, data science, neuroscience and design of intelligent algorithms and systems.
+
+## Why scona and neuroscience?
+
+Being initiative and enterprising, I am always willing to learn new things and look for new ideas. I value self-development and consider education to be the best investment. 
+Based on the Coursera’s recommendation once I’ve decided to complete the online course “Learning how to learn” (the best ever course) presented by Dr. Barbara Oakley. 
+This is how I have got excited about neuroscience field. What I liked the most, all the tips and suggestions of how to “learn smart not hard” were backed up by insightful 
+facts and recent researches, that inform about the way our brain works, which is helpful for improving one's productivity. 
+That’s why the chance to apply my knowledge and skills to develop the tool for neuroscientist was so appealing to me.
+
+From the list of INCF’s projects, Scona project grabbed my attention the most. It matches my previous experience and skills as well as my interests. 
```

Comment:
Add a couple of links. To the INCF list (maybe on neurostars?) and to scona?

Also, I think I always write scona in lowercase to match the package name. Would you mind changing it?

Reply:
Links are added.
From now on I will write "scona" in lowercase :)